app/src/main/java/de/danoeh/antennapod/fragment/PlaybackHistoryFragment.java

```diff
@@ -258,7 +127,7 @@ protected void afterBindViewHolder(EpisodeItemViewHolder holder, int pos) {
             // played items shouldn't be transparent for this fragment since, *all* items
             // in this fragment will, by definition, be played. So it serves no purpose and can make
             // it harder to read.
-            holder.itemView.setAlpha(1.0f);
+            holder.itemView.findViewById(R.id.container).setAlpha(1.0f);
```

Comment:
Hmm, this is a bit weird (like you already wrote). I will need to have a look at this.